cluster-autoscaler/config/migconfig.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+type MigConfig struct {
+	MinSize int
+	MaxSize int
+	Url     url.URL
+}
+
+type MigConfigFlag []MigConfig
+
+func (this *MigConfigFlag) String() string {
+	var b bytes.Buffer
+	b.WriteString("[")
+	for i, npc := range *this {
+		if i > 0 {
+			b.WriteString(" ")
+		}
+		b.WriteString(fmt.Sprintf("%d:%d:%s", npc.MinSize, npc.MaxSize, npc.Url.String()))
+	}
+	b.WriteString("]")
+	return b.String()
+}
+
+func (this *MigConfigFlag) Set(value string) error {
+	tokens := strings.SplitN(value, ":", 3)
+	if len(tokens) != 3 {
+		return fmt.Errorf("wrong nodes configuration: %s", value)
+	}
+	ncf := MigConfig{}
```

Comment:
Please don't use names such as `ncf`. I have no idea what this means. It makes this code much harder to read.